kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -131,6 +133,15 @@
       currentUserEdit: null,
     }),
     computed: {
+      LEARNER() {
```

Comment:
I like it. Possible to get even shorter:

```javascript
      LEARNER: () => UserKinds.LEARNER,
      COACH: () => UserKinds.COACH,
      ADMIN: () => UserKinds.ADMIN,
```
b8603b5beb1e72a14e38dbd06c37fd3fbaed7b7f